sigmastate/src/main/scala/sigmastate/interpreter/Interpreter.scala

```diff
@@ -170,11 +171,11 @@ trait Interpreter extends ScorexLogging {
 
   /** Extracts proposition for ErgoTree handing soft-fork condition.
     * @note soft-fork handler */
-  def propositionFromErgoTree(tree: ErgoTree, ctx: CTX): SigmaPropValue = {
+  def propositionFromErgoTree(tree: ErgoTree, validationSettings: SigmaValidationSettings): SigmaPropValue = {
```

Comment:
These changes are not required for this PR. It is better to keep the old code unchanged since this part of the code will need to be revised for cross version implementation (3.x + 4.0). Chances are the ValidationSettings will not be enough, so let's keep the old code which passes the whole context in the method. 
Please rollback.

Reply:
ok reverted